src/raw/mod.rs

```diff
@@ -531,7 +531,12 @@ impl<T, A: Allocator + Clone> RawTable<T, A> {
     #[inline]
     #[cfg(feature = "raw")]
     pub fn allocation_info(&self) -> (NonNull<u8>, Layout) {
-        self.table.allocation_info(Self::TABLE_LAYOUT)
```

Comment:
Can this check be moved into the method in `RawTableInner`?

Reply:
Are you sure? This way we will do branch (albeit trivial) for paths we don't use in normal operation.

At the moment I added assertion to `RawTableInner.allocation_info`.

Reply:
The reason for the split between `RawTable` and `RawTableInner` is to reduce the amount of LLVM IR generated and improve compilation times. In release builds, all the `RawTableInner` methods are inlined into the `RawTable` methods.

Reply:
Got it. Updated the PR.